### ISSUE TITLE
Review and fix our own Notify templates -- URLs in email notifications read out fully, need to match staging

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#943]
Fix issue in Notify's own notifications through GC Notify | GC Notification

Match and test on staging service first

With 6 long links in the 'Your Notify account | Votre compete Notification' each of them is read out fully -- consider embedding the link instead.

Notify's templates + Template formatting

Test other notifications that reach the public
https://trello.com/c/JvhSOLnS/148-inclusive-design-review-notifications

https://support.microsoft.com/en-us/office/make-your-outlook-email-accessible-to-people-with-disabilities-71ce71f4-7b15-4b7a-a2e3-cf91721bbacb

**Comment by anikbrazeau** [comment]
See template for "Notify password reset email" for example of how to embed link with description.

**Comment by mcman12** [comment]
Moving to next sprint. Need to talk with devs at Dev-design sync!

**Comment by anikbrazeau** [comment]
Test on staging first, otherwise use password reset as example of how to do this on Markdown

Document and share with those who have stake in links discussion --

**Comment by anikbrazeau** [comment]
Match staging and prod templates

**Comment by ClementineHahn** [comment]
Need full heads down for 2 hours to be done and was hard to get so far. Anik will try to do that today, should be done by the end of today. 
The french on those templates is good on prod

**Comment by ClementineHahn** [comment]
Work needed yesterday happened. Close to being done. 
Still work on MOU template needs to happen + make sure the feedback points to the "contact us" and not the support page. 
@brdunfield will work on this with @anikbrazeau
